source/retryable-writes/tests/README.rst

```diff
@@ -317,6 +317,10 @@ and sharded clusters.
     retryWrites=false to your connection string.
 
    and the error code is 20.
+   
+   **Note**: Drivers that rely on ``serverStatus`` to determine the storage engine
+   in use MAY skip this test for sharded clusters, since ``mongos`` does not report
+   this information in its ``serverStatus`` response.
```

Comment:
Opting for a MAY here, since drivers would certainly detect MMAPv1 through other means. Perhaps there is an environment variable that indicates what storage engine is being used throughout the sharded cluster. Alternatively, a driver could poke through and talk directly to the shard's primary to determine what storage engine is running.